repositories/tasks/file/csv/repository_test.go

```diff
@@ -0,0 +1,219 @@
+package csv // import github.com/mlimaloureiro/golog/repositories/tasks/file/csv
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math"
+	"os"
+	"regexp"
+	"testing"
+	"time"
+
+	tasksModel "github.com/mlimaloureiro/golog/models/tasks"
+	tasksRepositories "github.com/mlimaloureiro/golog/repositories/tasks"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	testCsvContent = `first-task,start,2019-01-01T10:00:00Z
+first-task,stop,2019-01-01T10:01:00Z
+second-task,start,2019-01-01T10:00:00Z
+second-task,stop,2019-01-01T10:04:00Z
+first-task,start,2019-01-01T10:02:00Z
+first-task,stop,2019-01-01T10:06:00Z
+last-task,start,2019-01-01T10:00:00Z
+`
+	otherTestCsvContent = `first-task,start,2010-06-01T15:00:00Z
+second-task,start,2019-01-01T10:00:00Z
+second-task,stop,2019-01-01T10:04:00Z
+last-task,start,2019-01-01T10:00:00Z
+`
+)
+
+func getFileContent(t *testing.T, filePath string) string {
+	stringBytes, err := ioutil.ReadFile(filePath)
+	assert.NoError(t, err)
+	return string(stringBytes)
```

Comment:
always add a new line in the last return, let's also add it to our styleguide

Reply:
Do you mean like this:
``` go
func getFileContent(t *testing.T, filePath string) string {
	stringBytes, err := ioutil.ReadFile(filePath)
	assert.NoError(t, err)
	
	return string(stringBytes)
}
````
?

When there is only one line this should be ignored right?
``` go
func GetPi() float64 {
    return 3.14
}
```